airbyte-integrations/base-python/airbyte_protocol/airbyte_protocol/__init__.py

```diff
@@ -1,4 +1,26 @@
 from typing import Generator
+import yaml
+import pkgutil
+import warnings
+import python_jsonschema_objects as pjs
+
+
+def _load_classes(yaml_path: str):
+    data = yaml.load(pkgutil.get_data(__name__, yaml_path), Loader=yaml.FullLoader)
+    builder = pjs.ObjectBuilder(data)
+    return builder.build_classes(standardize_names=False)
+
+with warnings.catch_warnings():
```

Comment:
is this because the underlying libraries output lots of noise? Are they something we'd want to retain/emit anywhere?

Reply:
There's one warning about the JSON schema version we're using. All of the features we're using here are supported, but the version itself isn't. Added a comment.